starcheck/src/starcheck.pl

```diff
@@ -659,8 +679,7 @@ sub json_obsids{
 my %save_hash;
 
 my $out = '<TABLE><TD><PRE> ';
-my $date = `date`;
-chomp $date;
+my $date = date_for_printing($RUN_START_TIME);
```

Comment:
Agreed and good point! (though I think we conflate happily in the ACIS checkers).  

I'd actually be inclined to skip adding anything new, as the thing we want, the seed time and temperature that is used for the thermal model, isn't available in this code.  I think we should just open a separate issue to actually save that out from calc_ccd_temps (because I don't think it is in either the return data from that tool or the saved-out json from the parent process.

Reply:
Skip adding anything new sounds good.